src/main/java/org/maxgamer/quickshop/listener/LockListener.java

```diff
@@ -107,6 +108,30 @@ public void onBreak(BlockBreakEvent e) {
         }
     }
 
+    /*
+     * Listens for sign placement to prevent placing sign for creating protection
+     */
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.LOWEST)
+    public void onSignPlace(BlockPlaceEvent event) {
+        Block placedBlock = event.getBlock();
+        if (!(placedBlock.getBlockData() instanceof WallSign)) {
```

Comment:
BlockLocker also can lock the container use a standing sign probably need to remove WallSign check.